configurations/pipeline_configuration.py

```diff
@@ -32,23 +33,25 @@
                     FlowResultConfiguration("RVI_elections_demog", "imaqal_pool_recently_displaced", "recently_displaced"),
 
                     FlowResultConfiguration("RVI_elections_s01e01_activation", "rqa_rvi_elections_s01e01", "rvi_elections_s01e01"),
+                    # FlowResultConfiguration("RVI_elections_s01e02_activation", "rqa_rvi_elections_s01e02", "rvi_elections_s01e02"),
```

Comment:
Since these are standard flows I suggest we create them and you uncomment this to avoid data delays in the subsequent weeks

Reply:
ok noted. Thanks.